### ISSUE TITLE
Predefinisani projekat 2

### ISSUE THREAD

**Issue by markosuhanov** [ftn-ai-lab/sc-2021-siit#51]
Student:
Marko Suhanov SW-99-2016

Asistent:
Dragan Vidaković

Metodologija: 
Za klasifikaciju vremenskih prilika u 4 klase (sunrise, cloudy, shine i rain) koristicu custom CNN model. Alat Tensorflow (Keras), metrika accuracy.

Repozitorijum:
https://github.com/markosuhanov/SoftKompjuting_Predef2
